From: David Bauer <mail@david-bauer.net>
Date: Sun, 31 May 2020 17:35:19 +0200
Subject: ath79-tiny: remove unused config symbols

These config symbols are only used for early models of the TL-WR940
series of wireless routers.

Also remove USB support, as this is not taken advantage of in Gluon.

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index 42243cfc481ab57f0b159535c8a3618c861070f9..740f5c1f6b650d36c7f2f85d09cf5250dad72afd 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -1,13 +1 @@
-CONFIG_LEDS_RESET=y
 CONFIG_MTD_SPI_NOR_USE_4K_SECTORS=y
-CONFIG_NET_DEVLINK=y
-CONFIG_NET_DSA=y
-CONFIG_NET_DSA_MV88E6060=y
-# CONFIG_NET_DSA_TAG_QCA is not set
-CONFIG_NET_DSA_TAG_TRAILER=y
-CONFIG_NET_SWITCHDEV=y
-CONFIG_PHYLINK=y
-CONFIG_PHY_AR7100_USB=y
-CONFIG_PHY_AR7200_USB=y
-CONFIG_REGULATOR=y
-CONFIG_REGULATOR_FIXED_VOLTAGE=y
